Added task description for zero-shot-image-classification (#387)
huggingface/hub-docs: tasks/src/zero-shot-image-classification/about.md

## About the Task

> Zero-shot image classification is a task in computer vision where the aim is to classify images into one of several pre-defined categories, without any prior training or knowledge of the categories.

Reviewer: ```suggestion
Zero-shot image classification is a computer vision task to classify images into one of several classes, without any prior training or knowledge of the classes. 
```

Author: @merveenoyan Thank you for your much needed guidance. It really helped me a lot as it was my first contribution slightly_smiling_face
